providers/dev-guide.md

```diff
@@ -264,6 +264,15 @@ The logic in this class has
 quite strong assumptions on the data being stored. It assumes that it has the tree structure and each entity contains its kids as a nested hash.
 If you are able to achieve that structure in the `refresh_parser.rb`, you are halfway done. Otherwise, good luck :]
 
+### Defining Inventory
+
+Basically, refresh process consists of Collector, Persister and Parser classes (in most providers).
+When Collector loads data from external provider (like AWS), Persister defines structure of Inventory by InventoryCollection objects.  
```

Comment:
Should be `When the Collector`, not sure `(like AWS)` adds anything I think any developer reading this knows what a provider is, how about `the Persister defines the inventory structure based on InventoryCollection objects`

Reply:
ok, I wrote `like AWS` because it seems there is unclear terminology, what provider is. 
One time provider is EMS in our database, another time AWS/Azure/etc. is meant.